### PR TITLE
Added TR.md file for Turkish podcasts and removed duplicated links from README.md

### PR DESCRIPTION
**Pull request by yusufduyar**
Here are the changes I have made 

- There was duplicated podcasts on the list. I have removed them. 
- And added languages of all podcasts. 
- There are great podcasts that I'm listening in Turkish. I have added them to the list.

### CODE REVIEW

**Comment by rShetty** [comment]
@yusufduyar Can we extract out non-english podcasts to a separate file ? Check #104

**Comment by yusufduyar** [comment]
@rShetty Ok I will add new TR.md file for Turkish podcasts.

**Comment by yusufduyar** [comment]
@rShetty I have added Turkish podcasts to TR.md file.

File: README.md [hunk 1]
```diff
@@ -140,11 +140,6 @@ List of podcasts which are helpful for software engineers/programmers.
   * **Description**: Data science, machine learning, artificial intelligence
   * **Frequency**: Weekly
 
-* [Machine Learning Guide](http://ocdevel.com/podcasts/machine-learning)
```
Comment (rShetty) [inline README.md]: Why did we remove this ?

File: README.md [hunk 2]
```diff
@@ -694,11 +689,6 @@ List of podcasts which are helpful for software engineers/programmers.
   * **Description**: Our mission is to provide concise and relevant daily briefings on the critical news happening across the global cyber security domain. In an industry overloaded with information, we also help individuals and organizations rapidly find the news and information that's important to them.
   * **Frequency** : Daily
 
-* [Darknet Diaries](https://defensivesecurity.org/)
```
Comment (rShetty) [inline README.md]: And this ?

Comment (rShetty) [inline README.md]: Was this duplicated ?

**Comment by yusufduyar** [comment]
All the links I have removed are have different name but same link. You can check them from Readme.md file.

**Comment by rShetty** [comment]
Cool thanks

**Pull request merged**